sdk/endpoints.go

```diff
@@ -312,11 +323,61 @@ func GetReceipt(delegateNode types.Node, hash string) (*types.Receipt, error) {
 	return &transaction.Receipt, nil
 }
 
+// GetTransactions - Get details about sent transactions for a node
+func GetTransactions(delegateNode types.Node, page, pageStart, pageSize string) ([]types.Transaction, error) {
```

Comment:
Ideally, page, pageStart, and pageSize would be optional (since the "easy" use case is that you don't care what they are).